### DISCUSSION TITLE
MSCDEX does not detect CD drive

### DISCUSSION THREAD

**Discussion by DSN9990** [86Box/86Box#1295] [Q&A] [answered]
**Describe the bug**
MSCDEX does not detect my CD drive on my DOS 5.0 machine, even with it set up correctly in the settings. I don't think this is a bug, but rather a incorrect AUTOEXEC.BAT configuration. I really need help.

**To Reproduce**
1. Install MSCDEX on a DOS 5 machine (I used a floppy disk image)
2. Reboot 
3. See error:
```
Device driver not found: 'MSCD001'.
No valid CDROM device drivers selected
```

**Expected behavior**
MSCDEX should detect CD drive.

**Screenshots**

![20210303-062526-428](https://user-images.githubusercontent.com/69558812/109763712-402cef00-7bd1-11eb-915b-849d0c630c57.png)

**Desktop (please complete the following information):**
 - OS: Windows 7
 - 86Box v3.0 build 2812 (533b7a5f)
 - Build type: Optimized Core2

**Additional context**
I tried also /D MSCD002, /D MSCD003 and it results on the same error above.

Config:

[86box config.zip](https://github.com/86Box/86Box/files/6074166/86box.config.zip)

HDDs:

[MS-DOS 5.0 486DX 33 MHz.zip](https://github.com/86Box/86Box/files/6074172/MS-DOS.5.0.486DX.33.MHz.zip)
[MS-DOS 5.0 486DX 33 MHz HDD 2.zip](https://github.com/86Box/86Box/files/6074226/MS-DOS.5.0.486DX.33.MHz.HDD.2.zip) (empty)

**Comment by dhrdlicka** [comment, marked as answer]
Your CD-ROM driver is actually configured as `MSCD000` in CONFIG.SYS:
```ini
DEVICEHIGH=C:\CDROM\GSCDROM.SYS /D:MSCD000 /v
```

**Comment by DSN9990** [comment]
Sorry, I was wrong. Thanks.
